src/utils/__test__/filter.test.ts

```diff
@@ -6,45 +6,60 @@ import { fsa } from '@chunkd/fs';
 import { asyncFilter } from '../chunk.js';
 
 describe('AsyncFilter', () => {
-  function makeGenerator(list: string[]): () => AsyncGenerator<{ path: string }> {
-    return async function* gen(): AsyncGenerator<{ path: string }> {
-      for (const path of list) yield { path };
+  function makeGenerator(paths: string[]): () => AsyncGenerator<{ url: URL }> {
+    return async function* gen(): AsyncGenerator<{ url: URL }> {
+      for (const path of paths) yield { url: new URL(`memory://${path}`) };
     };
   }
 
-  it('should include all', async () => {
-    const gen = makeGenerator(['a.tiff', 'b.tiff', 'c.tiff']);
-    const result = await fsa.toArray(asyncFilter(gen()));
-    assert.equal(result.length, 3);
-  });
+  // it('should include all', async () => {
```

Comment:
Is that meant to be removed?

Reply:
@blacha? This is commit 804b355b64496912bb9dabb62a2d33736a7c9023.

Reply:
it looks to have been a mistake it should be uncommented.